test/spawn.jl

```diff
@@ -469,3 +479,19 @@ let c = `ls -l "foo bar"`
     @test length(c) == 3
     @test eachindex(c) == 1:3
 end
+
+## Deadlock in spawning a cmd (#22832)
+# FIXME?
```

Comment:
Would test_broken have worked? A simple response to the question "it froze on CI and couldn't reproduce locally" would be a lot more visible than the merge commit text.